spec/controllers/home_page_controller_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe HomePageController, type: :controller do
   describe 'GET show' do
+    login_user
+
     it 'has a 200 status code' do
       get :show
       expect(response.status).to eq(200)
```

Comment:
@sinpai let's add one more test that will check that unlogged user gets `403` http status

Reply:
In my case, there is a redirect to a login page, which will give 302 http status. I've implemented test using native `expect(response).to redirect_to(new_user_session_path)` handler for such a case.